src/components/Amalgamation/AmalgamatingBusinesses.vue

```diff
@@ -486,11 +474,23 @@ export default class AmalgamatingBusinesses extends Mixins(AmalgamationMixin, Co
       return
     }
 
-    // Check for Legal API fetch issues.
+    // Check for business not in LEAR (Auth/Legal dbs).
+    if (business.authInfo?.status === 'NOT_FOUND') {
+      // Report error.
+      console.log('Missing auth info.')
+      this.showUnableToAddBusinessDialog()
+
+      // Hide spinner.
+      this.$root.$emit('showSpinner', false)
+
+      return
+    }
+
+    // Check for fetch issues.
     // NB - don't check for null firstTask since that's valid
-    if (!business.businessInfo || !business.addresses || !business.firstFiling) {
+    if (!business.authInfo || !business.businessInfo || !business.addresses || !business.firstFiling) {
       // Report error.
-      console.log('Missing business info or addresses or first filing.')
+      console.log('Missing auth info or business info or addresses or first filing.')
       this.showSomethingWentWrongDialog()
```

Comment:
"If any of the required objects is falsy (null) then there was some sort of network issueerror. Report error."

Reply:
Eg, search for any active business, then set network to Offline, then try to add it.

![image](https://github.com/bcgov/business-create-ui/assets/10002889/13f0ff49-9dd7-4bfa-b7c6-e7c4025e4032)